src/DotLiquid/StandardFilters.cs

```diff
@@ -227,6 +227,26 @@ public static string StripHtml(string input)
         }
 
         /// <summary>
+        /// Converts the input object into a formatted currency as specified by the culture info.
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="cultureInfo"></param>
+        /// <returns></returns>
+        public static string Currency(object input, string cultureInfo = "en-US")
+        {
+            decimal amount;
+
+            if (decimal.TryParse(input.ToString(), out amount))
+            {
+                var culture = new CultureInfo(cultureInfo);

```

Comment:
I am not a globalization expert, but I think in this case it is better to use `new CultureInfo(cultureInfo, false);` in order to avoid any side effect due to the culture of the machine, as stated in [MSDN](https://msdn.microsoft.com/en-us/library/ky2chs3h(v=vs.110).aspx#Anchor_2).